containers/podman-desktop
# feat: use micromark button directive in onboarding and remove button component (#3668)

### What does this PR do?

it removes the button component and allow to define a button using markdown syntax

### Screenshot/screencast of this PR

Nothing changes. This is the button rendered
![micromark_button](https://github.com/containers/podman-desktop/assets/49404737/3e8cf307-f35e-410a-9f5a-7ad1af86c108)

### What issues does this PR fix or reference?

it resolves #3668 

### How to test this PR?

1. start the onboarding by making one of the system requirements check to fail to see the button
